config/config.go

```diff
@@ -1032,6 +1044,9 @@ func (c *Cluster) ValidateExistingVPC(existingVPCCIDR string, existingSubnetCIDR
 	// Loop through all subnets
 	// Note: legacy instanceCIDR/availabilityZone stuff has already been marshalled into this format
 	for _, subnet := range c.Subnets {
+		if !subnet.Create {
```

Comment:
validation is still needed, instead of checking for IP ranges, check that subnet actually exist